modules/swagger-codegen/src/main/resources/swift/api.mustache

```diff
@@ -16,14 +16,26 @@ extension {{projectName}}API {
 /** {{description}} */{{/description}}
 public class {{classname}}: APIBase {
 {{#operation}}
+    {{#allParams}}
+    {{#isEnum}}
+    /**
+
+    enum for parameter {{paramName}}
+    */
+    public enum {{{datatypeWithEnum}}}_{{operationId}}: String { {{#allowableValues}}{{#values}}
```

Comment:
@cjolif I agree with your comment.

@fabdslv I've a pending PR (https://github.com/swagger-api/swagger-codegen/pull/2508) that should be able to handle the situation.